aws_cloudtrail_rules/aws_key_compromised.py

```diff
@@ -0,0 +1,14 @@
+def rule(event):
+    # Capture PutUserPolicy
+    request = event.get('requestParameters')
+    return event.get('eventName').startswith(
+        'PutUserPolicy') and request.get('policyName').startswith('AWSExposedCredentialPolicy_DO_NOT_REMOVE')
```

Comment:
Similar comment here on startswith(), should we just check that the string `AWSExposedCredentialPolicy` is in the `policyName` somewhere?

Reply:
made this an equality check as this is always the policy thats applied